### ISSUE TITLE
Error in spelling of chrystalMoved damage type (should be crystalMoved).

### ISSUE THREAD

**Issue by James103** [brandon3055/Draconic-Evolution#1448]
## Base information
* Minecraft version: 1.12.2 (and most likely much earlier given the commit dates)
* Mod version: Since ec1a113e655dd63a432bac99b65f3043babfd0e6 (in lang files) and bb3b128c2dda949deb4197f2758373ff14ff1a18 (in code)
* Minecraft Forge version: N/A
* Mod Pack: N/A

## The problem
There is an error in the spelling of the `chrystalMoved` damage type. The damage type should have been called `crystalMoved` (without the h). The string can be produced in chat by attempting to place down a Chaos Crystal somewhere.

Alternatively, you can use the following CraftTweaker scripts to inflict damage every time you do work on an anvil:
```
import crafttweaker.player.IPlayer;

// This produces the correct death message.
events.onPlayerAnvilRepair(function(event as crafttweaker.event.PlayerAnvilRepairEvent){
	if (event.entity instanceof IPlayer) {
		val player as IPlayer = event.entity;
		val test_damage_source = crafttweaker.damage.IDamageSource.createOfType("chrystalMoved").setDamageIsAbsolute().setDamageBypassesArmor();
		player.attackEntityFrom(test_damage_source, 9999.0);
	}
});
```

```
import crafttweaker.player.IPlayer;

// This produces an incorrect death message: "death.attack.crystalMoved'.
events.onPlayerAnvilRepair(function(event as crafttweaker.event.PlayerAnvilRepairEvent){
	if (event.entity instanceof IPlayer) {
		val player as IPlayer = event.entity;
		val test_damage_source = crafttweaker.damage.IDamageSource.createOfType("crystalMoved").setDamageIsAbsolute().setDamageBypassesArmor();
		player.attackEntityFrom(test_damage_source, 9999.0);
	}
});
```

**Comment by FoxMcloud5655** [comment]
Nice catch, but is it even worth it to fix at this point with the rewrite in-progress?

**Comment by brandon3055** [comment]
Fixed in 1.16. I never bothered to actually localize this because this is a message that _ideally_ no one should ever see.
